features/cache.feature

```diff
@@ -12,3 +12,18 @@ Feature: Cache
         Then the directory named "~/somewhere_else" should exist
         And a file named "~/somewhere_else/freshli.db" should exist
         And we can open a SQLite connection to "~/somewhere_else/freshli.db"
+
+    Scenario: Destroy
```

Comment:
Let's add a destroy scenario that uses an alternative cache directory location.

Reply:
I had tried, but I was having an unbelievable amount of trouble with the paths. After an hour, I gave up.

Reply:
I understand that it can be frustrating. But it still needs to be done.

Reply:
If you want a hand with it, please let me know.

Reply:
Added